lib/index.js

```diff
@@ -62,7 +62,7 @@ try {
 
   const triggeredBy = action?.comment?.user?.login || action?.sender.login;
 
-  const canRelease = allowedUsers.includes(triggeredBy);
+  const canRelease = allowedUsers.includes(triggeredBy) || allowedUsers.length === 0;
```

Comment:
Isn't this a bit of security issue? The allowed-users list is marked optional for easier install, however keeping it empty would mean anybody can trigger releases and that can be a bit problematic. I would add another input `unsafe-empty-release` so the person installing this and using it will know that it's unsafe.